ui/app/models/variable.js

```diff
@@ -80,4 +80,33 @@ export default class VariableModel extends Model {
       return acc;
     }, {});
   }
+
+  // Gets the path of the variable, and if it starts with jobs/, delimits on / and returns each part separately in an array
+
+  /**
+   * @typedef pathLinkedEntities
+   * @type {Object}
+   * @property {string} job
+   * @property {string} [group]
+   * @property {string} [task]
+   */
+
+  /**
+   * @type {pathLinkedEntities}
+   */
+  get pathLinkedEntities() {
+    const entityTypes = ['job', 'group', 'task'];
+    const emptyEntities = { job: '', group: '', task: '' };
+    if (this.path.startsWith('jobs/') && this.path.split('/').length <= 4) {
+      return this.path
+        .split('/')
+        .slice(1, 4)
+        .reduce((acc, pathPart, index) => {
+          acc[entityTypes[index]] = pathPart;
+          return acc;
+        }, emptyEntities);
+    } else {
+      return emptyEntities;
```

Comment:
`blocking`: I think this needs feature and programmer tests otherwise, I think this will break in the future.